data_structure.cpp

```diff
@@ -82,25 +83,26 @@ void ShippingFormList::addForm(ShippingForm* &Form) {
 	this->FormList.push_back(Form);
 }
 
-void ShippingFormList::inputList() {
+void ShippingFormList::inputList(std::ifstream &filein) {
 	char choice = 'N';
 	std::cout << "The current database has " << FormList.size() << " forms\n";
 	std::cout << "Do you want to add more forms? (Y/N) :";
-	std::cin >> choice;
+	//filein>>choice;
+	filein>> choice;
```

Comment:
Quy ước làm đẹp code là khi cin, cout thì viết cách toán tử 1 khoảng trắng

cin >> a >> b; //OK
cin>> a >> b; //cin>> thieu dau cach
cin >>a >> b; // >>a thieu dau cach